OpenEdXMobile/res/layout/learn_selection_item.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/tv_learn_item"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center_vertical"
+    android:padding="@dimen/edx_half_margin"
+    android:textColor="@color/primaryDarkColor"
+    android:textSize="@dimen/edx_medium" />
```

Comment:
> Text size should be 14sp or @dimen/edx_small

matched the font size with the design, cuz Figma files use `px` and android use `sp` for font sizes.

> Please add line-height as well: app:lineHeight="@dimen/widget_margin_double"

added.

> Padding is also a bit different from design but I think this will work 🤔

Figma files use `px` and android use `dp` for margin, and paddings.

> Plus the font and color are different on both these [designs](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=1114%3A1): Please coordinate with the design team.

will update the story description to match the decision with the design team.